packages/cisco_umbrella/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -164,7 +164,7 @@ processors:
       field: cisco.umbrella.identities
       separator: ","
       preserve_trailing: false
-      if: "ctx?.log?.file?.path.contains('dnslogs') && ctx?.cisco?.umbrella?.identities != null"
+      if: "(ctx?.log?.file?.path.contains('dnslogs') || ctx?.log?.file?.path.contains('proxylogs')) && ctx?.cisco?.umbrella?.identities != null"
```

```suggestion
      if: "ctx.cisco?.umbrella?.identities != null && (ctx.log?.file?.path.contains('dnslogs') || ctx.log?.file?.path.contains('proxylogs'))"
```
Moving cheaper test to the front and removing unnecessary null-safe operators.